docs/integration/nginx.md

```diff
@@ -1,12 +1,72 @@
 title: Nginx Monitoring Integration
 description: The Sematext monitoring agent can collect many metrics from NGINX instances, including requests, reading, writing, waiting, active and more. Utilize built-in anomaly detection, threshold, and heartbeat alerting and send notifications to email and various chatops messaging services. Correlate events & logs, filter metrics by server or time, and visualize your cluster's health with out of the box graphs and custom dashboards
 
-## Integration
+Sematext offers a simple Nginx monitoring agent, written entirely in Node.js without CPU and memory overhead. It's easy to install and require in your source code.
```

Comment:
written entirely in Node.js ==> axe, irrelevant, plus could change
without CPU and memory overhead ==> everything has _some_ overhead.  Maybe "minimal"?

Reply:
It's easy to install and require in your source code. -- there is no requiring for Nginx agent...